aea/cli/scaffold.py

```diff
@@ -55,6 +55,9 @@
 
 
 @click.group()
+@click.option(
+    "--local-package", is_flag=True, help="Scaffold skill outside a agent directory.",
```

Comment:
Can we rename this to `--to-local-registry` and short `-tlr`. and "Scaffold skill outside an agent directory to the local registry."